src/scripting/lua_common.cpp

```diff
@@ -590,12 +590,18 @@ bool luaW_totstring(lua_State *L, int index, t_string &str)
 t_string luaW_checktstring(lua_State *L, int index)
 {
 	t_string result;
-	if (!luaW_totstring(L, index, result))
-		luaW_type_error(L, index, "translatable string");
-	return result;
+	if(luaW_totstring(L, index, result)) {
+		return result;
+	}
+	size_t sz;
+	if(const char* cstr = lua_tolstring(L, index, &sz)) {
+		return std::string(cstr, sz);
```

Comment:
Is it okay for `luaW_checktstring()` to just accept regular strings like that?

Reply:
I'm not sure, but I don't _think_ it'll hurt anything?

Reply:
luaW_totstring should already accept them, so i wonder what this does?

Reply:
...ah, good point. I guess I should just remove this bit then.